eland/operations.py

```diff
@@ -517,11 +555,16 @@ def _unpack_metric_aggs(
 
                 # If this is a non-null timestamp field convert to a pd.Timestamp()
                 elif field.is_timestamp:
-                    agg_value = elasticsearch_date_to_pandas_date(
-                        agg_value, field.es_date_format
-                    )
+                    func = elasticsearch_date_to_pandas_date
```

Comment:
Aliasing a function like this is confusing, prefer to keep the function inline even if it's long.